tests/performance/bloom_filter.xml

```diff
@@ -1,10 +1,16 @@
 <test>
-
-
     <create_query>DROP TABLE IF EXISTS test_bf</create_query>
     <create_query>CREATE TABLE test_bf (`id` int, `ary` Array(String), INDEX idx_ary ary TYPE bloom_filter(0.01) GRANULARITY 8192) ENGINE = MergeTree() ORDER BY id</create_query>
+    <create_query>CREATE TABLE test_bf_indexOf (`id` int, `ary` Array(LowCardinality(Nullable(String))), INDEX idx_ary ary TYPE bloom_filter(0.01) GRANULARITY 1) ENGINE = MergeTree() ORDER BY id</create_query>
     <create_query>SYSTEM STOP MERGES</create_query>
-    <query>INSERT INTO test_bf SELECT number AS id, [CAST(id, 'String'), CAST(id + 1, 'String'), CAST(id + 2, 'String')] FROM numbers(1000000)</query>
+
+    <fill_query>INSERT INTO test_bf SELECT number AS id, [CAST(id, 'String'), CAST(id + 1, 'String'), CAST(id + 2, 'String')] FROM numbers(1000000)</fill_query>
+    <fill_query>INSERT INTO test_bf_indexOf SELECT number AS id, [CAST(id, 'String'), CAST(id + 1, 'String'), CAST(id + 2, 'String')] FROM numbers(1000000)</fill_query>
+
+    <query>SELECT count() FROM test_bf_indexOf WHERE indexOf(ary, '1') = 2</query>
```

Comment:
The modification for this test is completely wrong...
The original test was used to test the performance of INSERT.
It is executing the INSERT query arbitrary number of times.

You cannot test SELECT query on a table with unknown number of records.

Reply:
@alexey-milovidov 
I understood completely wrong.
But now I see that blocks of `<query>statement</query>`  are the targets of performance tests.
Thank you very much.

Reply:
@alexey-milovidov 
I appreciate that you completely fixed this codes. I will try to write source codes more precisely so that other people would be able to read my code easily.
I learned a lot from you. thank you.